apps/spark/src/test/java/com/linkedin/openhouse/jobs/spark/OperationsTest.java

```diff
@@ -339,6 +339,8 @@ public void testSnapshotsExpirationVersionsNoop() throws Exception {
     final String tableName = "db.test_es_versions_noop_java";
     final int numInserts = 3;
     final int versionsToKeep = 5; // Should keep all versions given that there are fewer versions
+    final int maxAge = 3;
```

Comment:
do we have a case where maxAge is not given?

Reply:
It's defined here: https://github.com/linkedin/openhouse/blob/7f66a730ec7d611335917e29bfed01050090c397/apps/spark/src/test/java/com/linkedin/openhouse/jobs/spark/OperationsTest.java#L275 when it's defined as 0. That being said if it is defined as 0 at a job-level it will fallback to the default of 3 days.

Reply:
Added additional tests when no configuration is defined

Reply:
yeah I think having a case with no config provided will be good. thx!